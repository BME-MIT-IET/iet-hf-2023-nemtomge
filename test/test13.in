loadmap map.json 
start V1 
set Scientist V1 PL1
set Gear PL1 Coat
show Gear V1
give Active V1 Stun 3 
touch V1 
0
show Gear V1
exit
