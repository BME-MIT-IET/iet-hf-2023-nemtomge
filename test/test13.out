Map from file map.json loaded
The game has been started. Players are: V1 
Scientist V1 set on field PL1 
Gear Coat has been placed on field PL1
V1 doesn't have any gears
Scientist V1 has been given active agent Stun time: 3  
Choose one to pick: 
0 - cancel
V1 doesn't have any gears
